doc/mediasdk-man.md

```diff
@@ -5739,7 +5739,7 @@ This structure specifies configurations for decoding, encoding and transcoding p
 `ICQQuality`            | This parameter is for Intelligent Constant Quality (ICQ) bitrate control algorithm. It is value in the 1…51 range, where 1 corresponds the best quality.
 `BufferSizeInKB`        | `BufferSizeInKB` represents the maximum possible size of any compressed frames.
 `NumSlice`              | Number of slices in each video frame; each slice contains one or more macro-block rows. If `NumSlice` equals zero, the encoder may choose any slice partitioning allowed by the codec standard. See also [mfxExtCodingOption2](#mfxExtCodingOption2)`::NumMbPerSlice`.
-`NumRefFrame`           | Number of reference frames; if `NumRefFrame = 0`, this parameter is not specified.
+`NumRefFrame`           | Max number of **all** available reference frames (for AVC/HEVC `NumRefFrame` defines DPB size); if `NumRefFrame = 0`, this parameter is not specified.<br><br>See also [mfxExtCodingOption3](#mfxExtCodingOption3)`::NumRefActiveP`, `NumRefActiveBL0` and `NumRefActiveBL1` which set a number of **active** references.
```

Comment:
I think, **DPB size minus 1** in terms of standard - in encoders we keep in DPB only **reference** frames so don't need to have extra +1 for output frame.
@dmitry-gurulev , @stasefremov , please correct me if I'm wrong.

Reply:
I would say that per standard (at least AVC) buffer for reconstruct of current frame isn't part of DPB. E.g. in DPB operation description from Annex C decoding/reconstruction process is apart from DPB modification. I.e. (1) frame is reconstructed (to some buffer, outside of DPB); (2) Then pictures from DPB are removed (to free space); (3) Reconstruct of current frame is added to DPB (occupying spare slot). See C.4.3. From this perspective I would say "defines DPB size".